### PR TITLE
Partial Template for Alignment Transition Viennese Bend

### PR DESCRIPTION
**Pull request by apinzenoehler**
Documentation requires improvement

### CODE REVIEW

**Comment by czapplitec** [comment]
For Viennese Bend, according to the fomula and all the test results, the LinearTerm is never used. I think it should be removed in this partial template.

File: IFC4x3/Templates/Partial Templates/Geometry/Curve Segment Geometry/Viennese Bend Transition Segment/DocTemplateDefinition.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocTemplateDefinition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Viennese_Bend_Transition_Segment_18qsFqvgTB0ftThvj4_r3E" Name="Viennese Bend Transition Segment" UniqueId="48d363f4-e6a7-4b02-9ddd-af9b44fb50ce" Type="IfcCurveSegment">
+	<Rules>
+		<DocModelRuleAttribute Name="ParentCurve">
+			<Rules>
+				<DocModelRuleEntity Name="IfcSeventhOrderPolynomialSpiral">
+					<Rules>
+						<DocModelRuleAttribute Name="ConstantTerm">
```
Comment (SergejMuhic) [inline IFC4x3/Templates/Partial Templates/Geometry/Curve Segment Geometry/Viennese Bend Transition Segment/DocTemplateDefinition.xml:8]: See https://github.com/bSI-InfraRoom/IFC-Specification/pull/129#issuecomment-873583205. Committed (ad585d9296421a44652c5510af48f54e2b6ee1a1) a correction proposal.

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
See ad585d9296421a44652c5510af48f54e2b6ee1a1

File: IFC4x3/Templates/Partial Templates/Geometry/Curve Segment Geometry/Viennese Bend Transition Segment/DocTemplateDefinition.xml (hunk 1 above)
Comment (SergejMuhic) [reply IFC4x3/Templates/Partial Templates/Geometry/Curve Segment Geometry/Viennese Bend Transition Segment/DocTemplateDefinition.xml:8]: @czapplitec please look at these amends.

Comment (czapplitec) [reply IFC4x3/Templates/Partial Templates/Geometry/Curve Segment Geometry/Viennese Bend Transition Segment/DocTemplateDefinition.xml:8]: Yes, I agree on this change

**Review by SergejMuhic — APPROVED** [review]

**Comment by SergejMuhic** [comment]
> 
> 
> I agree on this change.

I take this as a review approval. Merging. 😄

**Pull request merged**